### PR TITLE
Updated package.json

### PR DESCRIPTION
**Pull request by juampick**
@arzafran Could you please check the new `package.json` data? thanks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,37 @@
 {
   "name": "mern-seed",
   "license": "MIT",
+  "description": "The start point to rock with ReactJS and NodeJS.",
+  "homepage": "http://makingsense.com",
+  "keywords": [
+    "react",
+    "redux",
+    "node"
```
Comment (arzafran) [inline package.json]: add "sass" please

File: package.json [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+  ],
+  "author": "Making Sense's FE/BE/UI Development team <hello@makingsense.com>",
```
Comment (arzafran) [inline package.json]: FE/BE/UX please.

**Review by arzafran — COMMENTED** [review]
please review comments so Farza can approve.

File: package.json (hunk 1 above)
Comment (juampick) [inline package.json]: ok, thanks for the catch

File: package.json (hunk 2 above)
Comment (juampick) [inline package.json]: ;)

**Comment by arzafran** [comment]
approved

**Review by arzafran — APPROVED** [review]
approved

**Pull request merged**